bin/check-bundle-size.js

```diff
@@ -69,7 +69,7 @@ function isTooBig({ path, size }) {
     case 'html': return size >   1_200;
     case 'ico':  return size >  16_000;
     case 'js':   return size > 200_000;
-    case 'png':  return size >  40_000;
+    case 'png':  return size > 200_000;
```

Comment:
The first time I added the banner@2x image, it was about 111KB.  After running through pngout, it is about 80KB. This limit still needed to change, but maybe it doesn't need to be this high.